modules/candidate_parameters/jsx/monitoring.js

```diff
@@ -216,39 +216,40 @@ var MonitoringStatus = React.createClass(
       for (let row in this.state.Data.history) {
         if (this.state.Data.history.hasOwnProperty(row)) {
           var line = "";
+          var lineInfo = "";
```

Comment:
Use `const` for values that won't be reassigned or `objects`, `arrays` and `let` for everything else